packages/eds-core-react/src/components/Tabs/Tab.tsx

```diff
@@ -28,6 +29,7 @@ const StyledTab = styled.button.attrs<TabProps>(
     font-size: 1rem;
```

Comment:
Suggest cleaning up the css thats now replaced by typography template. Font-family, font-size.

Reply:
Done :)

Reply:
Done 😊